### ISSUE TITLE
Train from scratch on my own customized dataset.

### ISSUE THREAD

**Issue by h0t-zer0** [ZhengPeng7/BiRefNet#270]
I want to use this remarkable segmentation model to do segmentation tasks on my own customized datasets (4k-20k image-GT pairs, with binary GTs). I don't need to perform with a high resolution, a resolution of 512 should be OK. I read the paper and code, and found that a classification loss is used in the training process (which is not needed for my dataset). So what should I do to start a training from scratch? I'm looking forward to your reply!

**Comment by ZhengPeng7** [comment]
You can just ignore it. The classification was only employed in the very first version for the DIS task with label annotations.
Since the effects were not obvious, I've removed it from all settings afterwards.
So, you can either train from scratch or fine-tune with my pre-trained weights.
But now that you have 4k-20k images, if the images are a bit different from your cases (for example, your data is specifically for cars), you can just train from scratch.
Another thing, though the provided BiRefNet models were pre-trained in a higher resolution than the needed 512x512, they still benefit from the higher one in my own previous experiment of selecting swin backbones.

**Comment by h0t-zer0** [comment]
Thanks for your quick reply! I'll follow your YouTube and bilibili videos and try to start my training from scratch.
